app/assets/scss/_service_submission.scss

```diff
@@ -32,8 +32,8 @@
 
   margin-top: $gutter;
 
-  .button-destructive,
-  .button-save {
+  .govuk-button--warning,
+  .govuk-button {
```

Comment:
Instead of changing these and modifying govuk-frontend styling, how about either keeping these class names or rename them to something else and add those class names to the govukButton params.

For example, if we were to keep the class names the same:
```
{{ govukButton({
	"text": "Delete",
    "text": "Delete draft service",
    "classes": "govuk-button--warning button-destructive",
}) }}

```

The benefit is that we are not patching govuk-frontend and its less likely that if 
govuk-front changes the buttons it will not break our button styling

Reply:
Cool - have used your suggestion and amended the last commit, thanks Al 👍